src/task/MonitorTask.ts

```diff
@@ -274,7 +273,7 @@ export default class MonitorTask
       });
     }
 
-    if (this.isMeetingConnected) {
+    if (this.context.isSessionConnected) {
```

Comment:
Just switching this over to the context var for consistency. The issue with using the task local var is that race conditions due to waiting on, e.g. `audioVideoDidStop` will lead to error logs (or at least, did so for the Chime App).